htdocs/accountancy/class/lettering.class.php

```diff
@@ -745,6 +745,9 @@ public function getLinkedDocumentByGroup($document_ids, $doc_type)
 
 		if (empty($document_ids)) {
 			return array();
+		} else {
+			//remove empty and null parts of document_ids
+			$document_ids = array_filter($document_ids);
```

Comment:
array_filter need 2 parameter but second one is empty.
What does this function do when second parameter is null...

Reply:
hello @eldy , array_filter MAY have 2 parameters

https://www.php.net/array_filter

> "If no callback is supplied, all empty entries of array will be removed. See [empty()](https://www.php.net/manual/en/function.empty.php) for how PHP defines empty in this case."

and that is exactly what we need, the only warning is about zero value (wich will be removed too) but in that case we can't have zero values ...